src/Middleware/ConfigureNewrelicForRequest.php

```diff
@@ -50,7 +50,7 @@ private function nameRouteIfRouteFound(ServerRequestInterface $request): void
     /**
      * @throws MiddlewareOutOfOrderException
```

Comment:
**suggestion:** MiddlewareOutOfOrderException is raised? I think we can remove this  docblock